.github/workflows/build-and-push-bsky-aws.yaml

```diff
@@ -2,7 +2,7 @@ name: build-and-push-bsky-aws
 on:
   push:
     branches:
-      - main
+      - production
```

Comment:
With new workflow, packages will be published to npm on merge into `main`. To build containers and push to AWS/GHCR, we'll require a second step of PRing into the `production` branch.